src/book/03-guides/04-general/01-git.mdx

```diff
@@ -64,6 +66,28 @@ If your code is in a different repository and it uses C++ or protobuf files, use
 
 If you push your changes to GitHub and get a message that the build failed due to formatting issues, try running the relevant format commands, commit any changes, and push to GitHub again. This will trigger a new build, which will check the formatting again. You won't be able to merge the PR until the build checks pass.
 
+## Reviewing Code
+
+To review code on GitHub open up a PR and open the Files Changed tab. This will give you the diff of the PR. You can click on the lines of code in the diff to add a comment and/or a suggestion to those lines. You can either leave a single comment or start a review. In a review all comments you make will have pending written on them to say that they have not been posted yet. Please make each comment non-ambiguous and clearly state what your concern is.
+
+When reading the code, if there is anything that confuses you or think is incorrect make a comment with all your concerns. For example, if there is a nontrivial block of code without a comment explaining it you should ask for the author to provide such a comment. Another example is if there is a block of code that could do something that the autor may have not considered you should inform the author, with a comment, on what these other effects may be.
```

Comment:
Adding a sentence on how to review NUbook/documentation PRs somewhere is probably worthwhile IMO
```suggestion
When reading the code, if there is anything that confuses you or anything you think is incorrect, make a comment with all your concerns. For example, if there is a non-trivial block of code without a comment explaining it you should ask for the author to provide such a comment. Another example is if there is a block of code that could do something that the author may have not considered. In such a case, you should inform the author with a comment, on what these other effects may be. 

When reviewing documentation PRs, you should look for factual inaccuracies, omissions of important information, awkward sentences, and typos.
```